src/test/frameworks/nuxt/helpers.ts

```diff
@@ -0,0 +1,7 @@
+import type { NuxtOptions } from "../../../frameworks/nuxt/interfaces";
```

Comment:
Nit: this file seems unnecessary if its only being used in the spec file.

Reply:
@TheIronDev it's also being used here: https://github.com/firebase/firebase-tools/blob/a476540d2a908bfd5168e5342dfdf4c1a6c5040b/src/frameworks/nuxt/index.ts#L14

Reply:
@TheIronDev sorry, just realized you're not referring to `frameworks/nuxt/interfaces.ts` but to `test/frameworks/nuxt/helpers.ts`. I expected this file to have more stuff but it wasn't necessary. Thanks, removing it now.